### ISSUE TITLE
[PICO-PI-IMX7] No ethernet interface on hardknott_5.10.52-2.1.0-next

### ISSUE THREAD

**Issue by pino-kim** [TechNexion/meta-tn-imx-bsp#9]
**Description**
Cant not find an ethernet interface on the PICO-PI-IMX7 prebuilt image

**Steps to reproduce the issue:**
```
$ mkdir tn-imx-yocto && cd tn-imx-yocto
$ repo init -u https://github.com/TechNexion/tn-imx-yocto-manifest.git -b hardknott_5.10.y-next -m imx-5.10.52-2.1.0.xml
$ repo sync -j8
$ WIFI_FIRMWARE=y WIFI_MODULE=qca DISTRO=fsl-imx-xwayland MACHINE=pico-imx7 BASEBOARD=pi source tn-setup-release.sh -b build-fsl-imx-xwayland-pico-imx7
$ bitbake imx-image-multimedia
```

**Describe the results you received:**
Can not find **eth0** interface on PICO-PI-IMX7 prebuilt-image on the target device

**Additional information you deem important (e.g. issue happens only occasionally):**
![Captura de tela de 2021-12-21 13-50-46](https://user-images.githubusercontent.com/29390119/147112667-5ca261b1-d706-4962-b501-0a1c07080748.png)

**Comment by richard-hu** [comment]
@pino-kim 

Thanks for the feedback.

I will check it tomorrow.  :)

BR,

Richard

**Comment by pino-kim** [comment]
@richard-hu 
Oh, Thanks for the first reply.

Cheers ~~ !!

**Comment by johnweber** [comment]
@pino-kim - So that we can check the board version, can you please supply a picture of the module including the serial number?

**Comment by pino-kim** [comment]
Hi, @johnweber
 
Thanks to reply

>  So that we can check the board version, can you please supply a picture of the module including the serial number?

Really apologize to you
I can't understand what you mean 

Can you take a picture of an example?

**Comment by richard-hu** [comment]
@pino-kim 

I just solve this issue in Linux 5.10.72_2.2.0 branch.
https://github.com/TechNexion/linux-tn-imx/commits/tn-imx_5.10.72_2.2.0-next

The problem results from outdated defconfig which is from Linux 5.4.70 and not compatible with Linux 5.10.x .

I rebase and update tn_imx_defconfig. Now eth0 interface shows up. 

> repo init -u https://github.com/TechNexion/tn-imx-yocto-manifest.git -b hardknott_5.10.y-next -m imx-5.10.72-2.2.0.xml
> repo sync -j8 

I update the demo image as the link below. Could you give it a try?

https://download.technexion.com/.to_customer/imx-image-multimedia-pico-imx7-20211223104506.rootfs.wic.bz2

BR,

Richard

**Comment by pino-kim** [comment]
Oh, Thanks @richard-hu 

I will feedback soon

**Comment by pino-kim** [comment]
Hi @richard-hu, @johnweber  

Apology late reply

ethe0 interface is shown on branch 5.10.72_2.2.0-next

But I hope this issue fix in 5.10.52-2.1.0-next too ^^

Merry Christmas and happy new year
